guides/source/association_basics.md

```diff
@@ -1258,7 +1258,7 @@ Controls what happens to the associated object when its owner is destroyed:
 * `:destroy` causes the associated object to also be destroyed
 * `:delete` causes the associated object to be deleted directly from the database (so callbacks will not execute)
 * `:nullify` causes the foreign key to be set to `NULL`. Callbacks are not executed.
-* `:restrict_with_exception` causes an exception to be raised if there is an associated record
+* `:restrict_with_exception` causes an `ActiveRecord::DeleteRestrictionError` exception to be raised if there is an associated record
```

Comment:
Can you address L1662 too?

https://github.com/rails/rails/blob/d2c8e9b4efcbee5785dd4c54956254bdcfaa2b9c/guides/source/association_basics.md#L1662